### ISSUE TITLE
hosts file from Ansible Resource Model Source ignored when running a command

### ISSUE THREAD

**Issue by mkeller0815** [rundeck-plugins/ansible-plugin#150]
Hello,

I have installed rundeck version 2.9.3 running with the ansible-plugin 2.2.0. I have configured an "Ansible Resource Model" pointing to my hosts file. Hosts are showing up and facts are gathered.

I have also set the node executor for Ansible Ad-Hoc Node Executor. 

If I select a host from the node list and try to run a single command (like "uptime") I'll get a 
warning that only localhost is available. This warning seems to be generated by ansible. If I symlink my hosts file to the default location in /etc/ansible/hosts everything works fine. 
So it seems that the "run command" executed by the Ansible Ad-Hoc Node Executor does not use the inventory file configured in the Resource Model.

Here's my project config: 

```
project.ansible-become-method=sudo
project.ansible-become=true
project.ansible-executable=/bin/bash
project.ansible-ssh-auth-type=privateKey
project.description=
project.disable.executions=false
project.disable.schedule=false
project.jobs.gui.groupExpandLevel=1
project.name=TestAnsible
project.nodeCache.delay=300
project.nodeCache.enabled=true
project.ssh-authentication=privateKey
project.ssh-keypath=/home/ansible/.ssh/id_rsa
resources.source.1.config.ansible-become=false
resources.source.1.config.ansible-gather-facts=true
resources.source.1.config.ansible-ignore-errors=true
resources.source.1.config.ansible-inventory=/home/ansible/ansible/hosts
resources.source.1.type=com.batix.rundeck.plugins.AnsibleResourceModelSourceFactory
service.FileCopier.default.provider=com.batix.rundeck.plugins.AnsibleFileCopier
service.NodeExecutor.default.provider=com.batix.rundeck.plugins.AnsibleNodeExecutor
```

**Comment by DerekTBrown** [comment]
I think this is the same as https://github.com/Batix/rundeck-ansible-plugin/issues/147?

**Comment by mkeller0815** [comment]
It seems so. And there are other issues aiming the same thing. 
One comment says 
> try to add:
> -i /rundeck-ansible/hosts
> in the "Extra Ansible arguments"
But not were to add the argument. I did not found any "Extra Ansible arguments". 
I was also facing the problem that the plugin is not able to deal with ssh-keys that have a password set. I was able to overcome this by running the rundeck Service as user ansible with an active ssh-agent in the same shell and the keys added to it.
